tests/compliance_operator_v2_test.go

```diff
@@ -278,6 +278,62 @@ func TestComplianceV2DeleteComplianceScanConfigurations(t *testing.T) {
 	assert.Empty(t, scanconfigID)
 }
 
+func TestComplianceV2ComplianceObjectLabels(t *testing.T){
+	ctx := context.Background()
+	conn := centralgrpc.GRPCConnectionToCentral(t)
+	service := v2.NewComplianceScanConfigurationServiceClient(conn)
+	serviceCluster := v1.NewClustersServiceClient(conn)
+	clusters, err := serviceCluster.GetClusters(ctx, &v1.GetClustersRequest{})
+	assert.NoError(t, err)
+	clusterID := clusters.GetClusters()[0].GetId()
+	testName := fmt.Sprintf("test-%s", uuid.NewV4().String())
+	req := &v2.ComplianceScanConfiguration{
+		ScanName: testName,
+		Id:       "",
+		Clusters: []string{clusterID},
+		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
+			OneTimeScan: false,
+			Profiles:    []string{"rhcos4-moderate-rev-4"},
+			Description: "test config",
+			ScanSchedule: &v2.Schedule{
+				IntervalType: 1,
+				Hour:         15,
+				Minute:       0,
+				Interval: &v2.Schedule_DaysOfWeek_{
+					DaysOfWeek: &v2.Schedule_DaysOfWeek{
+						Days: []int32{1, 2, 3, 4, 5, 6},
+					},
+				},
+			},
+		},
+	}
+
+	resp, err := service.CreateComplianceScanConfiguration(ctx, req)
+	assert.NoError(t, err)
+	assert.Equal(t, req.GetScanName(), resp.GetScanName())
+
+	query := &v2.RawQuery{Query: ""}
+	scanConfigs, err := service.ListComplianceScanConfigurations(ctx, query)
+	configs := scanConfigs.GetConfigurations()
+	scanconfigID := getscanConfigID(testName, configs)
+	defer deleteScanConfig(ctx, scanconfigID, service)
```

Comment:
We do not wait for the scan to finish, it is deleted soon after the labels are checked.
This should be fine since the purpose is to check scan's labels.